### PR TITLE
feat(tests): fix Integration tests

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This PR brings back integration tests. It needs to be rebased once #13657 is merged

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/hardhat-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### 0.2.0
```
Comment (julien51) [inline packages/hardhat-plugin/CHANGELOG.md:3]: Please change the version in the package.json file too :)

File: packages/hardhat-plugin/test/index.test.ts [hunk 2]
```diff
@@ -34,13 +34,19 @@ describe('Unlock Hardhat plugin', function () {
...
+    })
+
     it('should store and extend existing networks info', function () {
       assert.isTrue(Object.keys(this.hre.unlock.networks).includes('1'))
       assert.equal(this.hre.unlock.networks['1'].name, networks['1'].name)
       assert.equal(this.hre.unlock.networks['1'].id, networks['1'].id)
       assert.equal(this.hre.unlock.networks['1'].unlockAddress, 'newAddress')
     })
     it('should store additional networks info', function () {
+      console.log(this.hre.unlock.networks['12345'])
```
Comment (julien51) [inline packages/hardhat-plugin/test/index.test.ts:48]:
```suggestion
```

**Review by julien51 — APPROVED** [review]
A few small nits/changes and then LG!

**Comment by clemsos** [comment]
Need to merge #13778 before this one

**Comment by clemsos** [comment]
Maybe good to make sure we are using latest versions of the contracts https://github.com/unlock-protocol/unlock/issues/13498

**Pull request merged**